paper.tex

```diff
@@ -307,41 +327,44 @@ \subsection*{Unambiguous low-level representation}
-    We should mention coasim~\citep{mailund2005coasim} because its model
-    seems quite close to what we have. It's not suitable as a standard though
-    because it uses a programming language. Besides coasim, there was
-    PopPlanner~\citep{ewing2010msms}
-    (a Java GUI which allows the user to build models
-    graphically and export the corresponding \ms\ command lines)
-    and MODELER4SIMCOAL2~\citep{antao2007modeler4simcoal2}
-    (a webapp that provided a set of common model templates that could be
-    parameterised to output the corresponding simcoal input files.)
-    \jkcomment{Not sure where these fit into the narrative, just
-    summarising here for later reference}.
-
-\end{itemize}
+The difficulty of describing complex demographic models
+for population genetic simulations has long been acknowledged
+~\citep[][e.g.]{antao2007modeler4simcoal2}, % jk - any older refs for this?
+and a number of different methods have been proposed to
+make such simulations more accessible and less error prone.
+Several Graphical User Interface (GUI) and visualisation
+methods have been developed, which greatly facilitate
+interpretation~\citep{mailund2005coasim,antao2007modeler4simcoal2,
+ewing2010msms,zhou2018popdemog}. However, these methods
+currently have little traction as they are all either directly coupled
+to an internal simulation method or to the specific syntax
+of a given simulator. Another approach that has been taken to
+simplify running such simulations is to provide an API in a
+scripting language such as Python or R, providing a flexible
+interface either to a built-in simulation
+engine~\citep{thornton2014cpp,kelleher2016efficient,haller2017flexible}
+% jk: are/were there other simulation frontends out there?
+or as a frontend for other simulators~\citep{staab2016coala}.
+While APIs have many advantages and have grown in popularity in
+recent years, the demographic model descriptions are tied
+to the details of both the underlying simulation engines
+and the programming language in question.
+
+Stable and healthy software ecosystems require standard interchange
+formats, allowing for the development of high-quality and long-lasting
+tools that produce and consume the standard.
+Demographic models are a key part of population genetics research,
+and to date the transfer of inferred models to downstream simulations
+has been ad-hoc, and conversions between the many different ways
+of expressing such models both labour intensive and fraught with errors.
+The proposed Demes standard is an attempt to bridge this gap
+between inference and simulation, and also to provide the foundations
+for a sustainable ecosystem of tools built around this data model.
+Table~\ref{tab-software} shows some initial infrastructure that we have
+built as part of developing Demes, but many other useful tools
+can be envisaged that either produce or consume this format.
+
+% Could finish up here by talking about reproducibility?
```

Comment:
```suggestion
A standard format also serves to improve scientific reproducibility. Authors simply include their demographic model as a supplementary file, or upload it to a data repository, so that readers need not transcribe the model themselves. This removes the opportunity for error due to model misinterpretation or incorrect translation to a simulatable format. In the event that the Demes standard becomes outdated, published Demes files will be convertible to future formats with relative ease because the data model is unambiguously specified.
```